### PR TITLE
added API content for energy/highlights

### PR DESCRIPTION
**Pull request by robert-bryson**
As per [request](https://gsa-tts.slack.com/archives/C2N85536E/p1652381987108239).

### CODE REVIEW

**Review by btylerburton — APPROVED** [review]
Whew that linting commit makes this look huge! 👍🏼

File: www.data.gov/energy/highlights/index.html [hunk 1]
```diff
@@ -314,6 +314,22 @@ <h2 class="entry-title">APIs from the Energy Information Administration</h2>
...
+                    <li>Hourly electricity operating data, including actual and forecast demand, net generation, and the power flowing between electric systems</li>
+                    <li>408,000 electricity series organized into 29,000 categories</li>
+                    <li>30,000 State Energy Data System series organized into 600 categories</li>
+                    <li>115,052 petroleum series and associated categories</li>
+                    <li>34,790 U.S. crude oil imports series and associated categories</li>
+                    <li>11,989 natural gas series and associated categories</li>
+                    <li>132,331 coal series and associated categories</li>
+                    <li>3,872 Short-Term Energy Outlook series and associated categories</li>
+                    <li>368,466 Annual Energy Outlook series and associated categories</li>
+                    <li>92,836 international energy series</li>
```
Comment (nickumia-reisys) [inline www.data.gov/energy/highlights/index.html:452]: I thought `<li>` always had to be in either `<ol>` or `<ul>` for it to be good html, any reason it's not?

Comment (btylerburton) [reply www.data.gov/energy/highlights/index.html:452]: good catch, nick!
https://stackoverflow.com/questions/52717274/does-li-always-need-to-be-nested-between-ul-and-ol-not-nav

Comment (btylerburton) [reply www.data.gov/energy/highlights/index.html:452]: Tl;DR - yes

**Pull request merged**